src/accordion/accordion.spec.ts

```diff
@@ -271,15 +272,34 @@ describe('ngb-accordion', () => {
          expectOpenPanels(fixture.nativeElement, [false, false, false]);
        });
      })));
+
+  it('should override the type in accordion with type in panel', async(inject([TestComponentBuilder], (tcb) => {
```

Comment:
This is _very_ light on tests.... We need some more:
* what happens if no type is specified? (No binding to `[type]` on accordion / panel)?
* what happens if type is specified only on accordion level and no panel has type?

I know that those might sound like trivial cases but we _need_ to have them covered.

BTW: you don't have to re-use the same html in each and every test, feel free to have test-specific markup.